crystalBox.py

```diff
@@ -307,6 +309,37 @@ def getEquivalents(self, hkl):
         print(sg)
         print(pg)
         print("The equivalents of reflection", hkl, "include the following:", equivalents)    
+    
+    def makeCrystal(self):
```

Comment:
General comment on all the functions. I would add a 1-2 line description under each function to explain what it is trying to do.

Reply:
(I haven't done this consistently either...but I think it's a good thing to aspire to...)